server/src/main/java/no/ntnu/klubbhuset/domain/Order.java

```diff
@@ -4,9 +4,11 @@
 
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 @Entity
 @Data
+@Table(name = "VIPPSORDERS")
```

Comment:
Should this really be commited in this pull request?

Reply:
Nope. Den har lurt seg med, men har ikke så mye å si